### DISCUSSION TITLE
Welcome to BigFloat Discussions!

### DISCUSSION THREAD

**Discussion by bumbitzu** [bumbitzu/BigFloat#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sidr1234** [comment]
Hi! Thanks for class, it is useful.
But there is an unhanled exception for string, lets's say "-10000000.756530": "stoi argument out of range".

**Reply by bumbitzu** [reply]
Thank you for the feedback, this class indeed does not work well, anyway I will try to upload a repository with a new BigFloat class.

**Reply by bumbitzu** [reply]
Take a look at my new repository [XFloat-class](https://github.com/bumbitzu/XFloat-class)
